src/engines/kotor/creature.cpp

```diff
@@ -327,8 +341,7 @@ void Creature::createPC(CharacterGenerationInfo *info) {
 	}
 
 	// set the specific class to level 1
-	_levels.clear();
-	_levels.emplace_back();
+	_levels.resize(1);
```

Comment:
I think you merged the emplace_back() fix into the wrong commit here.

https://github.com/xoreos/xoreos/pull/208/commits/6fd4e0ad9d621ea2ecc2f8c3d34dfb425f9c17b8#diff-d2c774ad01449ab50752df0719218776R331 adds it, and you remove it here. You probably meant to merge that fix into 6fd4e0ad9d621ea2ecc2f8c3d34dfb425f9c17b8 instead.